pom.xml

```diff
@@ -425,7 +425,6 @@ THE SOFTWARE.
           <!-- Version specified in parent POM -->
           <configuration>
             <consoleOutput>true</consoleOutput>
-            <failsOnError>true</failsOnError>
```

Comment:
This is for the Checkstyle plugin, right? Would be nice to retain this behavior for non-CI builds, e.g. via a default profile

Reply:
> This is for the Checkstyle plugin, right?

Yes.

> Would be nice to retain this behavior for non-CI builds

The existing behavior _is_ retained for non-CI builds via [`<failOnViolation>`](https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#failOnViolation), which defaults to true. `<failsOnError>` is unnecessary to retain the existing behavior.